tuf/requests_fetcher.py

```diff
@@ -97,7 +98,7 @@ def chunks():
           # Before beginning the round, sleep (if set) for a short amount of
           # time so that the CPU is not hogged in the while loop.
           if settings.SLEEP_BEFORE_ROUND:
-            time.sleep(settings.SLEEP_BEFORE_ROUND)
+            time.sleep(settings.SLEEP_BEFORE_ROUND)  # type: ignore[unreachable]
```

Comment:
Here I received the error:
```
tuf/requests_fetcher.py:101: error: Statement is unreachable  [unreachable]
```
mypy warns that it's unreachable, because `settings.SLEEP_BEFORE_ROUND` is `None`, but in real
application that could be changed and that's why this warning is unnecessary.

Reply:
if we wanted to annotate the old code base, the fix for this would be to annotate SLEEP_BEFORE_ROUND correctly... 

but I would rather not do that (and instead remove the tuf/requests_fetcher.py changes from this PR)